version/version.go

```diff
@@ -18,6 +18,8 @@ var (
 	Version = "unknown"
 	// Build timestamp
 	BuildTime = time.Now().String()
+	// Git branch that was used to build the binary
+	GitBranch = ""
```

Comment:
Let's update the LogBuildInformation() method below to include this (if not empty)

Reply:
done